fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -1867,6 +1869,21 @@ class ShardManager {
 		dirtyShards->insert(getMetaDataShard());
 	}
 
+	void logWriteSetMetrics() {
+		double duration = now() - lastWriteSetLogTime;
+		cumulativeWriteSet.insert(writeSet.begin(), writeSet.end());
+		TraceEvent(SevInfo, "WriteSetStats", this->logId)
+		    .detail("WriteSetSize", writeSet.size())
+		    .detail("CumulativeWriteSetSize", cumulativeWriteSet.size())
+		    .detail("duration", duration);
+		if (duration > 7200) {
```

Comment:
Nits: do we want to add a knob to the 7200? to make it tunable?

Reply:
Just curios, does counter not work here?